README.md

```diff
@@ -1,2 +1,43 @@
+| **A Launch property**  | A tag property  | Use the singular version of 'tag' when writing "tag {noun}" in a sentence. | When users install an extension to an Adobe Experience Platform **Launch property**, they will be shown the extension configuration view which your extension will provide.  | When users install an extension to **a tag property**, they will be shown the extension configuration view which your extension will provide.  |
+| **the {noun} extension in Platform Launch**  |  the {noun} tag extension in Platform | The name of the extension will precede the tag qualifier. Later references to tags (Launch) are removed once context has been established.  | Learn about **the Adobe ContextHub extension in Adobe Experience Platform Launch**. | Learn about **the Adobe ContextHub tag extension in Adobe Experience Platform**.  |
+| **Adobe Experience Platform Launch {verb} ...**  | Adobe Experience Platform {verb} | Launch is not always necessary and can be omitted. When the sentence refers to Launch as a service it can often be replaced by referring to Platform. | **Adobe Experience Platform Launch provides** a unified catalog where users can view tag extensions that are available for installation. | **Adobe Experience Platform provides** a unified catalog where users can view tag extensions that are available for installation. |
+| **An edge extension in Launch** | an edge extension OR a tag extension in an edge property  | An edge extension is not to be confused with an edge configuration. Using 'an edge extension' is fine for general use. This example only applies if more context is required.  | Learn how to define an action-type library module for an **edge extension in Adobe Experience Platform Launch.** | Learn how to define an action-type library module for **a tag extension in an edge property**.  |
+
+The following table provides an example of event forwarding branding guidelines. It is handled in the same way as tags in that it is a suite of technologies and not a Branded service. It is referred to in lower case.
+
+| Original Term  |Updated Term | Explanation  |Previous Example Sentence  | Current Branded Use |
+|---|---|---|---|---|
+| **the Server Side Cloud Connector extension**  | the Cloud Connector event forwarding  extension	| It is an 'event forwarding extension', the name of the extension will precede it. Also, as context is established by 'event forwarding', 'Launch can be removed'.  | Learn about **the Server Side Cloud Connector extension** in Adobe Experience Platform Launch.  |  Learn about **the Cloud Connector event forwarding extension** in Adobe Experience Platform. |
+
```

Comment:
I don't think "event forwarding" has enough examples of nuanced rules that it needs its own section. This would be better incorporated into a section about how to refer to extensions (tag extensions or event-forwarding extensions, as mentioned previously).